app/src/main/java/chat/rocket/android/util/Extensions.kt

```diff
@@ -36,6 +44,16 @@ var TextView.textContent: String
         text = value
     }
 
+var TextView.content: CharSequence
+    get() = text
+    set(value) {
+        Markwon.unscheduleDrawables(this)
```

Comment:
Do we need this for all TextViews? Maybe a simple version for TextViews, and another property for markdown texts.

Reply:
No, we don't need actually. I could rename the property from `content` to `markdown` if you will. We have already a simpler version which is `textContent` but it uses `String` as its type. Actually I don't know what's the real benefit of having this one. I think we could have stick just with the framework's `text` property.

Reply:
I think we should have a simple `content` with `CharSequence` and a `markdown`, just that